routes/web.auth.php

```diff
@@ -33,6 +33,7 @@
 Route::middleware('participant')->group(function () {
     Route::get('profile/edit-camp/{event}', 'ProfileController@editCamp');
     Route::put('profile/edit-camp/{event}', 'ProfileController@editCampSave');
+    Route::get('pay/{event}', 'ProfileController@setupExistingPayment');
```

Comment:
Why not use a full reference like `[ProfileController::class, 'setupExistingPayment']`